lib/features/vpc/widgets/vpc_intro_item_family.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class VPCIntroItemFamily extends StatelessWidget {
+  const VPCIntroItemFamily({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox.expand(
+      child: Stack(
+        children: [
+          const Positioned.fill(
+            child: Text(
+              'We’ve made it easy for your children to take part in giving.\n\nIf you have multiple children, set up all your child profiles now. If you come out of the app you will need to give verifiable permission again.',
+              textAlign: TextAlign.center,
+              style: TextStyle(
+                fontSize: 17,
+                color: Color(0xFF184869),
+              ),
```

Comment:
AppTheme and `Theme.of(context).textTheme.bodyMedium!`

Reply:
fixed